### ISSUE TITLE
cardDlg does not work anymore ?

### ISSUE THREAD

**Issue by fab64443** [octgn/OCTGN#1646]
OCTGN version 3.1.293.0
Python API reference 3.1.02

I am trying to popup a dialog window with cardDlg as referenced in the python api reference :

```
def addMastermind(group, x=0, y=0):
    if shared.SetupDeck.controller != me:
        return
    cardlist = [ c for c in shared.SetupDeck if c.Type in ('Mastermind') ]
    dlg = cardDlg(cardlist)
    cardselected = dlg.show()
    cardselected.moveTo(shared.MastermindDeck)
```

And an error says that the cardDlg is not defined. 

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\User\Documents\Octgn\GameDatabase\024e796a-e222-4bdf-96ae-0196749de3be\scripts\actions.py", line 162, in addMastermind
NameError: global name 'cardDlg' is not defined

Am I doing something wrong ?

Sincerely,

F

**Comment by brine** [comment]
cardDlg is only available in the most recent python API version (3.1.0.2), double check that your game is on that version.

**Comment by fab64443** [comment]
Arrgh.  You're right. The game definition was in 3.1.0.1...
Thanks for your help.
